### PR TITLE
Make npm install quietly

### PR DESCRIPTION
**Pull request by easternbloc**
Mostly just moving things around.

Added a spinner too.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -77,23 +78,32 @@ var run = function bramble (args) {
...
                             }
                         }, function (err) {
+                            installSpinner.stop();
                             console.log(('NPM install failed').red);
                             throw err;
                         })
                         .fail(function () {
-                            console.log(('NPM tests failed restoring to old versions ' + previousPackages.toString().replace(',', ', ')).red);
-                            return npm.install(previousPackages, installOptions).fail(function (err) { throw err; });
+                            installSpinner.stop();
```
Comment (roc) [inline index.js]: isn't there a `fin` or always on Q? Could you put `installSpinner.stop()` calls there pls?

Comment (easternbloc) [inline index.js]: I knew you were gonna say that. You're like the voice in the back of my head :speak_no_evil:

Comment (roc) [inline index.js]: Does the voice at the back of your head unnecessarily repeat itself as well? :fire:

Comment (easternbloc) [inline index.js]: He never shuts up.

File: index.js [hunk 2]
```diff
@@ -88,24 +91,32 @@ var run = function bramble (args) {
                                 return new Q();
                             }
                         }, function (err) {
-                            console.log(('NPM install failed').red);
+                            process.stdout.write(('\nNPM install failed').red);
                             throw err;
                         })
+                        .fin(function () {
```
Comment (roc) [inline index.js]: :wave: hiya

Comment (easternbloc) [inline index.js]: :bow:

**Comment by roc** [comment]
Let's have at it.

I'm assuming this is fast moving to pure npm CLI usage in a 2.x release

**Comment by easternbloc** [comment]
:+1:

**Pull request merged**